examples/with-mongodb/next.config.js

```diff
@@ -1,8 +1,8 @@
+require('dotenv').config() /* For local environment var configuration */
```

Comment:
This next.config.js is no longer needed.

Reply:
Thanks!